### DISCUSSION TITLE
V5 如何在设置默认节点情况下，绘制一个其他类型节点

### DISCUSSION THREAD

**Discussion by MusicTen** [antvis/G6#6091] [Q&A]
1. 默认节点是circle
2. 特殊节点是rect
3. 数据中设置type，但还是以默认节点绘制的
4. 默认节点有些样式及状态，所以定义在node中

[复现连接](https://stackblitz.com/edit/react-mc86kp?file=index.js)

**Comment by Aarebecca** [comment]
node.type 的优先级大于 data.nodes[number].type ，你可以将 node.type 指定为：
```
type: (d) => d.type || 'circle',
```
